### PR TITLE
revisiting gmail import (redirection url + confirmation page

### PR DESCRIPTION
**Pull request by martinraison**
@stkem I'll plug the extension into this next

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/views/website/importContactsResult.scala.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+@content = {
+<main>
+    <h1 class="main-logo">kifi</h1>
+    <section>
+        <h3>
+            @if(isSuccess){
+                @if(numImported.isEmpty){Your}else{@{numImported.get}} gmail contacts were successfully imported!
+            } else {
+                Failed to import your gmail contacts.
```
Comment (stkem) [inline kifi-backend/modules/shoebox/app/views/website/importContactsResult.scala.html]: Maybe add "Please try again later" or something encouraging like that :-)

**Comment by stkem** [comment]
lgtm. Sorry for the delay.

File: kifi-backend/modules/shoebox/app/views/website/importContactsResult.scala.html (hunk 1 above)
Comment (2is10) [inline kifi-backend/modules/shoebox/app/views/website/importContactsResult.scala.html]: We should consistently use a capital “G” in “Gmail”, as Google does.

File: kifi-backend/modules/shoebox/conf/shoebox.routes [hunk 2]
```diff
@@ -57,6 +57,9 @@ GET     /admin/unsubscribe          @com.keepit.controllers.website.EmailOptOutC
 GET     /extmsg/email/mute          @com.keepit.controllers.website.EmailNonUserMuteController.optOut(publicId: String)
 POST    /extmsg/email/mute          @com.keepit.controllers.website.EmailNonUserMuteController.optOutAction(publicId: String)
 
+GET     /importContactsSuccess      @com.keepit.controllers.website.ContactsImportController.importContactsSuccess(redirectUrl: Option[String], numContacts: Option[Int])
```
Comment (2is10) [inline kifi-backend/modules/shoebox/conf/shoebox.routes]: Consider using RESTful-style routes. For example, all contacts-related URLs could go under `/contacts`:

```
/contacts/import
/contacts/import/success
/contacts/import/failure
/contacts/refresh
```

This helps with keeping routes organized logically in the routes file and, when the URLs are visible in the address bar, makes for a better user experience.
http://www.nngroup.com/articles/url-as-ui/
http://alanhogan.com/url-as-ui

Comment (martinraison) [inline kifi-backend/modules/shoebox/conf/shoebox.routes]: Thanks for the links! I just modified the routes

**Pull request merged**